config/config.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 var (
-	patJobName    = regexp.MustCompile(`^[a-zA-Z_][a-zA-Z0-9_-]*$`)
+	patJobName    = regexp.MustCompile(`^[a-zA-Z0-9_][a-zA-Z0-9_-]*$`)
```

Comment:
Is there a "Done" button or something for you to click here? The PR is now stuck on GitHub in the "Changes requested" state, which is part of their new review features they introduced recently...